draft-ietf-shmoo-remote-fee.md

```diff
@@ -104,9 +104,19 @@ there should always be a free option to make the process as open as possible. Ha
 it is of course strongly anticipated that at least all working group sessions as well as BoFs
 and the administrative plenary of an IETF meeting provide an option for remote participation.
 
-Further, in order to fully remove barriers to participation, any free
+Moreover, in order to fully remove barriers to participation, any free
 registration option must offer the same degree of interactivity and
-functionality available to paid remote attendees.  The free option
+functionality available to paid remote participants. Specifically, it must not
+be possible to identify participants that used the free option. However,
+of course this does not mean that all services must be provided for free to
+participants using the free registration option, but only those services
+that are provided as part of the regular registration. Offering additional
```

Comment:
I don't think so. The service should be usually offered to all participants and just because you don't use it, it doesn't mean you use the free option. I wrote below to "subset or all participants" because I was thinking, that you would e.g. offer sign language support only to deaf people, or live translation in a certain language only to those that speak that language. The intention was not to restrict additional paid services to people using the paid registration. E.g. even if you use the free participation option you might have sponsorship to paid for the sign language support. Or maybe I can just remove the "subset or all participants" part as it probably doesn't hurt to offer additional services to all participants in general even if the expectation is that only a subgroup would be able to make use of them....?

Reply:
Thought experiment: The IETF begins to offer a paid AR/VR remote participation service. A remote participants says to another - free-option-using - participant "we're having this really interesting meeting in this AR/VR space, join us".

Reply:
Also as I paid participants I don't think I could just go ahead and book this extra service without checking with my management and I think I would need have a good reason to justify this additional costs. So, "sorry I cant't paid for it" is a valid answer no matter what your "base" registration is.

Reply:
Also if suddenly all but the free participants start using this service, we should really add it to the "base" registration instead of offering it as "additional" service. This is what the last sentence tries to say.

However, we don't have this service now. So this all is very speculative... I personally don't think we need to say anything beyond what's already in the draft but this PR tries to address the discuss ballot.

Reply:
I don't think so and that's not the intention. Actually the intention of this text is exactly the opposite: there should not be a special tier for free participants but they will always get the normal "base" registrations. However, you can still  book additional services on top of that. This would be more like you can also book a social ticket or register for the hackathon after you have registered/during your registration. I guess the social is actually a good example. Okay, there is no remote option for the social, but if there were, this text says that you don't have to provide a free option for the social. (However, the last sentence still tries to say that one should anyway consider if it would be valuable to provide also a free option for inclusivity).

How can we make that more clear?

Reply:
I would just leave it at "it should not be possible to identify participants that used the free option" and remove everything else.

Reply:
Fine with me but I don't think this will address Rob's discuss, so please check with him/engage in the email discussion on his discuss.

Reply:
@rgwilton?

Reply:
I'm happy with the standard IETF remote service being free, but I think that it wrong to constrain the LLC to not optionally be able to offer additional paid services to a subset of remote participants, just because I see it as an unnecessary constraint.  If a significant proportion of participants were to make use of this optional paid service then I would hope that the LLC would look at including it in the default remote participation option, i.e., the community should trust the LLC to do the right thing.

Hence, generally, I would rather see guidance to the LLC about what the community wants them to achieve rather than hard "must" constraints which restricts how they can act.  Yes, in theory the LLC can ignore what is set out in this document, but I think that it would be wrong, and arguably harmful to its reputation and the IETF community, for it to do so.

If we were do the same thing for in-person, and say that there should always be an in-person fee waver option then having a similar constraint may raise problems:
 - would it mean that if you get a fee-waiver then you should also have free access to the social event?  And if so, would having a fee-waiver have to guarantee you a place at the social event?  I think that a reasonable proportion of the community might regard that as being unfair.

I would offer to move to abstain, but given that 4 other ADs are currently abstaining, that would effectively block this document, which given this is a process document is somewhat concerning to me anyway.

Reply:
Rob, I don't think there is anything in the draft that would restrict the LLC to offer additional services, even without this text. This new text only tries to clarify but not change or add anything that this document requires. So from my point of view I don't think that having or not having this text changes much....

Reply:
@rgwilton maybe to answer your question: no, this document does not require to give people who use a fee waiver also free access to the social. The social is an additional service/event that is not needed to visit to full participate in the IETF process; social tickets are usually limit or at least not all participants usually decide to go, so I don't think that would be an issue for inclusivity. However, the important part is that this document does not require this or add any restriction to offer these kind of things at an additional charge. And if there would be a desire in the community to also provide any such service for free (or integrate such service in the regular registration fee) that would be a separate discussion to have.